plugins/imuxsock/imuxsock.c

```diff
@@ -173,6 +176,11 @@ static int nfd = 1; /* number of active unix sockets  (socket 0 is always reserv
 			socket, even if it is not enabled. */
 static int sd_fds = 0;			/* number of systemd activated sockets */
 
+#if (defined(__FreeBSD__) && (__FreeBSD_version >= 1200061))
+	#define DFLT_bUseSpecialParser 1
+#else
+	#define DFLT_bUseSpecialParser 0
```

Comment:
Inverting these, setting DFLT_bUseSpecialParser to 0 in the FreeBSD case, seem to solve the issue.

This is the output from running rsyslogd with debug enabled.
```
3709.314145803:main thread    : imuxsock.c: imuxsock: allocating memory for 2 listeners
3709.314154565:main thread    : ratelimit.c: ratelimit:imuxsock:new ratelimiter:bReduceRepeatMsgs 0
3709.314273893:main thread    : imuxsock.c: imuxsock: set listener[1].bUseSpecialParser to 0 (default 0)
3709.314280628:main thread    : ratelimit.c: ratelimit:imuxsock:new ratelimiter:bReduceRepeatMsgs 0
3709.314287006:main thread    : imuxsock.c: imuxsock: set listener[2].bUseSpecialParser to 0 (default 0)
3709.314478218:main thread    : imuxsock.c: imuxsock: Opened UNIX socket '/var/run/log' (fd 3).
3709.314637005:main thread    : imuxsock.c: imuxsock: Opened UNIX socket '/var/run/logpriv' (fd 4).
```

Reply:
> Inverting these, setting DFLT_bUseSpecialParser to 0 in the FreeBSD case, seem to solve the issue.

yes, of course - made the same mistake twice ;-) I'll fix and then merge.

Reply:
Sounds good.

Reply:
Thank you for helping out and fixing this!